### DISCUSSION TITLE
Multiple errors when attempting to edit or create an app

### DISCUSSION THREAD

**Discussion by Cnote43** [Budibase/budibase#10964] [Help] [answered]
I set up a simple daily tracking app as a way of learning budibase about a year ago. I've been using the app daily since then and it's worked fine. A few days ago I decided I wanted to get back to learning budibase but I now find that when I attempt to open/edit/manage/create an app from the apps tab I get multiple errors, being some combination of:

- Error initializing app:
- Failed to sync with production database
- Error getting org config
- Error loading apps and templates

The app still seems to work fine, in that I can go to the published form and submit a new entry with no errors, but I can't edit the app or view the data through the UI.

I'm running the all-in-one container on Unraid using the "latest" tag, which has me on version 2.7.21. I've attached a log where I restarted the container, logged in, and then clicked on a published app to produce the errors. I'm not seeing anything in the logs reflecting a problem, but I might be missing it or maybe I need to change the logging level.

I haven't had any luck searching for solutions the past few days so I'm hoping someone here has some thoughts on how I might fix this.

[budibase_log.txt](https://github.com/Budibase/budibase/files/11793031/budibase_log.txt)

**Comment by benaunt** [comment, marked as answer]
We had this issue and followed this advice to fix it:

> Can you go to your settings -> organisation and toggle on analytics if its toggled off. 
>  
> We currently have an open issue based around this here: https://github.com/Budibase/budibase/issues/10924. Its currently in-progress and should be fixed swiftly.

 
Thanks

**Reply by Cnote43** [reply]
Thank you SO MUCH! it worked. No way I would have ever figured that out.
